repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss

```diff
@@ -0,0 +1,120 @@
+.blog-author {
+    margin:0px;
+}
+
+.blog-backstory {
+    background-color: #E5E9EC;
+    padding: 20px;
+}
+
+.blog-contactus {
+    padding: 20px;
+    margin: 20px 0 40px ;
+    border-left: 5px solid #CDCBCE;
+}
+
+.blog-social {
+    display: flex;
+
+    .blog-social-icon {
+        padding: 5px 5px 5px 0px;
+        flex: none;
+
+        &__img {
+            width:80px;
+        }
+    }
+}
+
+.lead-paragraph p {
+    font-size: 22px;
+}
+
+.lead-image-container {
+    width: 100%;
+    margin-bottom: 30px;
+
+    .lead-image-caption {
+        padding: 5px 2px 5px 2px;
+        background-color: white;
+    }
+}
+
+
+.blog-authors {
+
+    margin-top: 20px;
+    border-top: 1px solid $section-dividers;
+
+    &__item {
+        display: flex;
+        margin-top:40px;
+    }
+
+    &__icon-round {
+        margin-right: 3rem;
+        flex: none;
+
+        &__img {
+            border-radius: 50%;
+            width: 200px;
+            height: 200px;
+            margin-top: 20px;
+        }
+
+        &__nhsimg {
+            width: 200px;
+            margin-top: 20px;
+        }
+    }
+
+    &__title {
+        font-size: $h5-font-size;
+        font-weight: 600;
+        margin-bottom: 1.1rem;
+    }
+}
+
+
+.latestBlog {
+
+    padding-top:30px;
+    border-top: 1px solid $section-dividers;
+
+    &__item {
+        display: flex;
+        margin-top:40px;
+
+        &:not(:last-child) {
+            margin-bottom: $baseline-grid-unit * 6;
+        }
+    }
+
+    &__icon {
+        margin-right: 1.5rem;
+        width: 300px;
+        flex: none;
+
+        &__img {
+            width: 300px;
+            height: 200px;
```

Comment:
I fixing width and height good idea? I had similar problem in BlogHub and talked to Owain about it. He said we could fix width and crop on height. I implemented it that way using 'object-fit: cover' css. Is it worth to consider similar solution here?

Reply:
Good suggestion.  Will update.